api/apps/api/src/modules/clone/infra/export/mark-export-as-failed.handler.ts

```diff
@@ -29,7 +29,7 @@ export class MarkExportAsFailedHandler
     await this.apiEvents.createIfNotExists({
       kind,
       topic: resourceId.value,
-      externalId: kind + exportId,
+      externalId: kind + exportId, //TODO: Check this
```

Comment:
I'd leave this as todo, though in principle we may end up concatenating these the other way round (`exportId + '/' + kind`)... probably irrelevant in practice, but this way they look "sortable" when looking at them. No need to do anything now, in any case.